### ISSUE TITLE
ipad does not see msfs but iphone does

### ISSUE THREAD

**Issue by davemay135** [astenlund/fs2ff#77]
Occasionally I can see msfs as a device on my ipad pro v15.5 and I can usually see it on my iphone, but most of the time it won't display as a device in FF even though the IP for my FF is correct in FS2FF.  You mentioned making sure a port was not blocked.  How do I ensure that?

**Comment by astenlund** [comment]
Try to see if you have Local Network enabled on the iPad (Settings -> Privacy -> Local Network).

**Comment by davemay135** [comment]
Yes, it is.  ForeFlight is selected.  Like I said, it sometimes works, it seems that if I'm careful about when I open FS2FF and the tablet.  Is there a way to do it where it's not so tempermental?

**Comment by davemay135** [comment]
While I don't see FS2FF on my iPad, it's working just fine on my iPhone.
